packages/components/psammead-paragraph/README.md

```diff
@@ -0,0 +1,43 @@
+# @BBC/Psammead-Paragraph [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Paragraph)
```

Comment:
This should be updated to match https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-headings/README.md

Reply:
Fixed this now.